project/backend/database/models.py

```diff
@@ -120,5 +152,26 @@ class Proof(models.Model):
     node = models.ForeignKey(Node, on_delete=models.CASCADE, related_name="proofs")
 
 
-# class Question(models.Model):
-#     node = models.ForeignKey(Node,models.CASCADE)
+class Question(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Add a explicit id field named question_id with AutoField as primary key. Makes things easier.

Reply:
i thought django's built-in id configuration can be used. when you don't set any primary key field explicitly, django itself ads an id field. it can be used as question.id and i think this way it is easier. is it still necessary to implement it as question_id explicitly?

Reply:
django initializes it as 'id'. I think it is confusing.

Reply:
Well, whatever.